Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Editor/TexturePresetSelectionWidget.ui

```diff
@@ -2,6 +2,12 @@
 <ui version="4.0">
     <class>TexturePresetSelectionWidget</class>
     <widget class="QWidget" name="TexturePresetSelectionWidget">
+        <property name="maximumSize">
+            <size>
+                <width>16777215</width>
+                <height>160</height>
```

Comment:
These are interesting maximumSize defaults - I'm curious why these width and heights values are used.

Reply:
16777215 is the default "unlimited" value from Qt Creator, as of 160 it's because I couldn't find another way to make the tag selection widgets not take way too much space for the whole window.